ccdaservice/unservice.js

```diff
@@ -0,0 +1,18 @@
+var isWin = /^win/.test(process.platform);
+if( isWin ){
+	var Service = require('node-windows').Service;
```

Comment:
I tried sudo node unservice and wasn't able to remove the service (it was started via background service); I can kill it with kill -9 if needed, though. Is there a way to turn off the background service within OpenEMR if, for example, make it inactive in globals?